cargo-bolero/src/project.rs

```diff
@@ -48,18 +48,26 @@ pub struct Project {
     /// Build the standard library with the provided configuration
     #[structopt(long)]
     build_std: bool,
+
+    /// Fake using a nightly toolchain while using the default toolchain by using RUSTC_BOOTSTRAP
+    #[structopt(long)]
+    fake_nightly_toolchain: bool,
```

Comment:
Why not name this the same as what we're doing so it's clear and we don't have to document the mapping?

```suggestion
    rustc_bootstrap: bool,
```

Reply:
Just checking if you have opinions on this naming? I can without any issues rename the flag to `--rustc-bootstrap` in this PR, but don't know how you feel about my explanation above 😅

Reply:
I think I still prefer to go with rustc bootstrap. It's already [public](https://github.com/rust-lang/cargo/issues/6627) and I would prefer to inherit any available documentation about what it does. Make sense?

Reply:
The link you sent is to an issue that was actually closed as "won't do without an RFC" :sweat_smile: 

That said I have no strong opinions here, so I've just force-pushed a rename to `rustc_bootstrap` :)